MapboxNavigation/NavigationMapView.swift

```diff
@@ -156,6 +153,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     open var tracksUserCourse: Bool = false {
         didSet {
+            showsUserLocation = !tracksUserCourse
```

Comment:
It should be possible for both `showsUserLocation` and `showsUserCourse` (née `tracksUserCourse`) to be false, so that nothing is overlaid atop the map:

```swift
if tracksUserCourse {
    showsUserLocation = false
}
```